### PR TITLE
Update results.md

### PR DESCRIPTION
**Pull request by FoggyFinder**
fixed some typos

---

Just curious why don't create some general function for easier testing, I mean something like this:

```fsharp
let printRequset = 
    validateRequest >>
    function
    | Ok req -> printfn "My request was valid! Name: %s Email %s" req.Name req.Email
    | Error e -> printfn "Error: %s" e

```

and if I may to ask - why  `validateRequest` take Result<Request> as a parameter? 

```fsharp
let validateRequest reqResult =
    reqResult 
    |> Result.bind validateName
```

I suppose it could be just a Request

```fsharp
let validateRequest reqResult =
    reqResult 
    |> validateName
    |> Result.bind validateEmail
```

### CODE REVIEW

File: docs/fsharp/language-reference/results.md [hunk 1]
```diff
@@ -67,16 +67,16 @@ let validateRequest reqResult =
 let test() = 
     // Now, create a Request and pattern match on the result.
     let req1 = { Name = "Phillip"; Email = "phillip@contoso.biz" }
-    let res1 = validateRequest (OK req1)
+    let res1 = validateRequest (Ok req1)
     match res1 with
-    | Ok req -> printfn "My request was valid! Name: %s Email %s" req1.Name req1.Email
+    | Ok req -> printfn "My request was valid! Name: %s Email %s" req.Name req.Email
```
Comment (cartermp) [inline docs/fsharp/language-reference/results.md]: This should stay the same - `req` isn't defined anywhere

File: docs/fsharp/language-reference/results.md [hunk 2]
```diff
@@ -67,16 +67,16 @@ let validateRequest reqResult =
...
     | Error e -> printfn "Error: %s" e
-    // Prints " "My request was valid!  Name: Phillip Email: phillip@consoto.biz"
+    // Prints: "My request was valid!  Name: Phillip Email: phillip@consoto.biz"
 
     let req2 = { Name = "Phillip"; Email = "phillip@bananas.com" }
-    let res2 = validateRequest (OK req2)
+    let res2 = validateRequest (Ok req2)
     match res2 with
-    | Ok req -> printfn "My request was valid! Name: %s Email %s" req1.Name req1.Email
+    | Ok req -> printfn "My request was valid! Name: %s Email %s" req.Name req.Email
```
Comment (cartermp) [inline docs/fsharp/language-reference/results.md]: Same here, though the fix is to change `req1` to `req2`.

**Review by cartermp — CHANGES_REQUESTED** [review]
Thanks @FoggyFinder!

Changes look good modulo the things I noted.

Having a `printRequest` function is a good idea, and I'd certainly welcome the contribution.

As for writing code in a more point-free style, we want to keep that out of the code samples largely because their use typically comes with clever utilization of partial application, which we don't want people spending too much time wondering about when reading a code sample. Perfectly valid code in an IDE, or in a scenario where it's a huge advantage (e.g., dependency injection), but for code samples that don't have hover tooltips that allow you to explore the type signatures, it can become more difficult to follow.

File: docs/fsharp/language-reference/results.md (hunk 1 above)
Comment (FoggyFinder) [reply docs/fsharp/language-reference/results.md]: @cartermp there is a pattern matching, right?
`| Ok req -> ...`

Comment (cartermp) [reply docs/fsharp/language-reference/results.md]: Ah, for some reason I couldn't see what was going on. I blame that on a long day.

**Review by cartermp — APPROVED** [review]

**Pull request merged**